### PR TITLE
Add `listinstantouts` command

### PR DESCRIPTION
**Pull request by sputn1ck**
This PR adds the `listinstantouts` cli command which let's users monitor their instant out swaps.

### CODE REVIEW

**Review by hieblmi — COMMENTED** [review]
Was thinking that we could provide getter methods for the exposed fields rather than making them public.

File: instantout/actions.go [hunk 1]
```diff
@@ -179,13 +179,13 @@ func (f *FSM) InitInstantOutAction(eventCtx fsm.EventContext) fsm.EventType {
 		protocolVersion:  ProtocolVersionFullReservation,
 		initiationHeight: initCtx.initationHeight,
 		outgoingChanSet:  initCtx.outgoingChanSet,
-		cltvExpiry:       initCtx.cltvExpiry,
+		CltvExpiry:       initCtx.cltvExpiry,
```
[the remark this answers is not in the document]
Comment (bhandras) [reply instantout/actions.go:188]: I'd tend to agree, but given that the added use-case doesn't involve sharing state as all instantouts are fetched from the DB, we can keep the exported members too.

**Review by bhandras — APPROVED** [review]
LGTM 🎉

**Review by hieblmi — APPROVED** [review]
LGTM then 💾

**Pull request merged**